src/home/App.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import SignUn from "./Signup.component";
```

Comment:
#### I suggest to do like this here `import Signin from './Signin';` 

* The one you used is not working at all, and it is making the app to crash